src/elements/wordpress/wordpress.wc.svelte

```diff
@@ -0,0 +1,210 @@
+
+  const deploymentStore = window.configs?.deploymentStore;
+  const currentDeployment = window.configs?.currentDeploymentStore;
+
+  $: disabled = ((loading || !data.valid) && !(success || failed)) || invalid || !profile || status !== "valid" || selectCapacity.invalid || isInvalid([...fields]); // prettier-ignore
+
+  let message: string;
+  let modalData: object;
+
+  const tabs: ITab[] = [{ label: "Config", value: "config" }];
+  let fields: IFormField[] = [
+    {
+      label: "Name",
+      symbol: "name",
+      placeholder: "Wordpress Instance Name",
+      type: "text",
+      validator: validateName,
+      invalid: false,
+    },
+    {
+      label: "Username",
+      symbol: "adminUsername",
+      placeholder: "Admin Username, will be used in all authentications on the machine",
+      type: "text",
+      validator: validateName,
+      invalid: false,
+    },
+    {
+      label: "Password",
+      symbol: "adminPassword",
+      placeholder: "Admin Password, will be used in all authentications on the machine",
+      type: "password",
+      validator: validateRequiredPassword,
+      invalid: false,
+    },
+    {
+      label: "Email",
+      symbol: "adminEmail",
+      placeholder: "support@example.com",
+      type: "text",
+      validator: validateRequiredEmail,
+      invalid: true,
+    },
+  ];
+  // define this solution packages
+  let selectCapacity = new SelectCapacityUpdate();
+  const packages: IPackage[] = [
+    { name: "Minimum", cpu: 1, memory: 1024 * 2, diskSize: 10 },
+    { name: "Standard", cpu: 2, memory: 1024 * 2, diskSize: 50 },
+    { name: "Recommended", cpu: 4, memory: 1024 * 4, diskSize: 100 },
+  ];
+
+  async function deployWordpressHandler() {
+    loading = true;
+    success = false;
+    failed = false;
+    message = undefined;
+
+    if (!hasEnoughBalance()) {
+      failed = true;
+      loading = false;
+      message = "No enough balance to execute! Transaction requires 2 TFT at least in your wallet.";
+      return;
+    }
+
+    deployWordpress(data, profile, gateway)
+      .then((data: any) => {
+        modalData = data.deploymentInfo;
+        deploymentStore.set();
+        success = true;
+      })
+      .catch((err: string) => {
+        failed = true;
+        message = normalizeDeploymentErrorMessage(err, "Wordpress");
+      })
+      .finally(() => {
+        loading = false;
+      });
+  }
+  $: logs = $currentDeployment;
+</script>
+
+<SelectProfile
+  on:profile={({ detail }) => {
+    profile = detail;
+  }}
+/>
+<div style="padding: 15px;">
+  <form class="box" on:submit|preventDefault={deployWordpressHandler}>
+    <h4 class="is-size-4 mb-4">Deploy a Wordpress Instance</h4>
+    <p>
+      WordPress is the simplest, most popular way to create your own website or blog. In fact, WordPress powers over
+      43.3% of all the websites on the Internet.
+      <a target="_blank" href="https://manual.grid.tf/weblets/weblets_wordpress.html"> Quick start documentation</a>
```

Comment:
Manual link does not work.

Reply:
waiting for the merge in https://github.com/threefoldtech/info_grid/pull/3